src/lib/Microsoft.Health.Fhir.Ingest.Template/ILineInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Ingest.Template
+{
+    public interface ILineInfo
+    {
+        public int LineNumber { get; set; }
+
+        public int LinePosition { get; set; }
+
+        public bool HasLineInfo();
```

Comment:
If hasLineInfo = false, what are the values of LineNumber and LinePosition? Both zero?